### DISCUSSION TITLE
How to get affine or transformation field that has been performed?

### DISCUSSION THREAD

**Discussion by kvczx** [TorchIO-project/torchio#1155] [Q&A]
Hi there:
My current project is to do deep learning for landmarks recognition. 
Our landmarks have been marked and saved on the physical coordinate system of the original image and used as supervising data to train the network. 
If I use torchio to perform random morphological transformation on the image, how should I obtain the affine matrix (or deformation field) that has been performed on the image, to map the original feature points to the new image?
Sorry if this question is too basic, and thank you for your patience.

Also:
#1003
I found this discussion in the IDEA block, and I wonder if this feature is still not available?

**Comment by romainVala** [comment]
Hello
Sorry but nothing change regarding this topic, so I would give the same answer as in the post you pointed ...

Only for the affine matrix you can easily retrive: you take the image affine after the transform divided by the image affine before the transform.
